doc/contribute/guidelines.rst

```diff
@@ -881,8 +881,7 @@ attention needed and it will be ready for merge sooner than later:
    is green
 
 #. If you get request for changes and submit a change to address them, make
-   sure you click the "Re-request review" button on the GitHub UI to notify
-   those who asked for the changes
+   sure to notify those who asked for the changes
```

Comment:
Isn't "re-request review" the simplest way of notifying those who asked for the changes? Even though it hides it from the reviewer list, it's the standard GitHub way of doing this. Authors should look at the review summary at the bottom of the PR instead?
Some people filter notifications by type, and re-review requested could be one of them.
EDIT: I see @kartben and @henrikbrixandersen already mentioned this, I agree with them.

Reply:
I agree.